Add support for sync revoke of ticket

glzhao opened the pull request:
When revoke a ticket, we always need wait a long time (may be
DEFAULT_TICKET_EXPIRY sometimes) now. Revoke time can be greatly
shortened with this patch.

glzhao commented:
Hi,
    I have updated the branch, you can pull it now, thanks :)

jjzhang commented:
Thanks a lot;) One nitpicking suggestion is it might be better that defining OP_START_LEASE and OP_STOP_LEASE in paxos_lease.c rather than paxos_lease.h.

glzhao committed:
Add support for sync revoke of ticket

When revoke a ticket, we always need wait a long time (may be
DEFAULT_TICKET_EXPIRY sometimes) now. Revoke time can be greatly
shortened with this patch.

Signed-off-by: Guangliang Zhao <gzhao@suse.com>
